lib/ngx/log.md

```diff
@@ -0,0 +1,231 @@
+Name
+====
+
+`ngx.log` - manage the nginx error log for OpenResty/ngx_lua.
+
+Table of Contents
+=================
+
+* [Name](#name)
+* [Status](#status)
+* [Synopsis](#synopsis)
+    * [Intercept nginx error logs with specified log level](#intercept-nginx-error-logs-with-specified-log-level)
+* [Methods](#methods)
+    * [set_errlog_filter](#set_errlog_filter)
+    * [get_errlog](#get_errlog)
+* [Community](#community)
+    * [English Mailing List](#english-mailing-list)
+    * [Chinese Mailing List](#chinese-mailing-list)
+* [Bugs and Patches](#bugs-and-patches)
+* [Author](#author)
+* [Copyright and License](#copyright-and-license)
+* [See Also](#see-also)
+
+Status
+======
+
+This Lua module is currently considered experimental.
+The API is still in flux and may change in the future without notice.
+
+Synopsis
+========
+
+Intercept nginx error logs with specified log level
+-----------------------------------------
```

Comment:
Style: the dashes should at least as long as the previous line. Please fix other similar places. Thank you.